### ISSUE TITLE
Data: integrate list of subspecies and varieties

### ISSUE THREAD

**Issue by jnga** [newfs/gobotany-app#231]
(Brought over from Iteration 26 Google Doc.)

> > Tristram’s list of subspecies and varieties needs to be uploaded if not already done – see his email.
> 
> We need an example of where these are missing on the site. Ensure his data are complete and refreshed (he has finished full-key taxa as well).

Will the result of this become a part of the existing Access/CSV data? Or is there something new that needs to be done from a development standpoint? (Waiting on clarification)
